src/handlers/dashboard/get_chart_data.py

```diff
@@ -92,13 +97,20 @@ def _format_payload(df: pandas.DataFrame, query_params: dict, filters: list) ->
     payload["totalCount"] = int(df["cnt"].sum())
     if "stratifier" in query_params.keys():
         payload["stratifier"] = query_params["stratifier"]
+        counts = {}
+        for unique_val in df[query_params["column"]]:
```

Comment:
Why does the stratifier loop below use `.unique()` but this doesn't?

Reply:
the stratifier label is a one-> many relationship to the column, which in practice means it shows up a bunch of time in the dataframe.

Reply:
Should we throw a .unique() on here too though?

Reply:
no, because here we're summing on the column values, which gives us a single output row.

Reply:
Yeah I guess you wouldn't get the wrong answer, you'd just re-calculate it multiple times.

Reply:
ah! i see what you're saying now, sorry